README.md

```diff
@@ -25,6 +25,11 @@ upload content to https://gist.github.com/.
 
     pkg install gist
 
+<200c>For Ubuntu/Debian
```

Comment:
@ConradIrwin somehow this zero width Unicode character got introduced here. I'll remove it.